### ISSUE TITLE
How do I get device string with ScaleText for current value?

### ISSUE THREAD

**Issue by alexbk66** [HomeSeer/Plugin-SDK#184]
I have features with ScaleText, how can I get the current device string?

Both EProperty.Status and hsDevice.Status return empty string. Do I have to manually convert current device value to string?

**Comment by spudwebb** [comment]
As discussed here:  https://forums.homeseer.com/forum/developer-support/hs4-development-private/hs4-plugin-development/1464385-how-do-i-get-device-string-with-scaletext-for-current-value, we need a replacement for  CAPIGetStatus in the PluginSdk.
Tracked as  PSDK-144

**Comment by alexbk66** [comment]
I'm using HS3 IHSApplication for things not ported to HS4 (i.e. WriteHTMLImage, Event_Info, etc) - but some users report exception from hs3.APIVersion
```
HS3 APIVersion: System.Exception: Client is not connected to the server.
bij HSCF.Communication.Scs.Client.ScsClientBase.SendMessage(IScs Message message)
bij HSCF.Communication.ScsServices.Communication.RequestReplyMes senger`1.SendMessageAndWaitForResponse(IScsMessage message, Int32 timeoutMilliseconds)
bij HSCF.Communication.ScsServices.Communication.RequestReplyMes senger`1.SendMessageAndWaitForResponse(IScsMessage message)
bij HSCF.Communication.ScsServices.Communication.RemoteInvokePro xy`2.Invoke(IMessage msg)
bij System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(Mess ageData& msgData, Int32 type)
bij HomeSeerAPI.IHSApplication.get_APIVersion()
bij PluginLib.PluginBase.get_HS3()
```

**Comment by jldubz** [comment]
This was addressed in [v1.4.0.0](https://homeseer.github.io/Plugin-SDK-Docs/release_notes/pluginsdk_releasenotes_1_4_0_0.html)

[AbstractHsDevice.Image](https://homeseer.github.io/Plugin-SDK-Docs/api/HomeSeer.PluginSdk.Devices.AbstractHsDevice.html#HomeSeer_PluginSdk_Devices_AbstractHsDevice_Image), [AbstractHsDevice.StatusString](https://homeseer.github.io/Plugin-SDK-Docs/api/HomeSeer.PluginSdk.Devices.AbstractHsDevice.html#HomeSeer_PluginSdk_Devices_AbstractHsDevice_StatusString), and [AbstractHsDevice.DisplayedStatus](https://homeseer.github.io/Plugin-SDK-Docs/api/HomeSeer.PluginSdk.Devices.AbstractHsDevice.html#HomeSeer_PluginSdk_Devices_AbstractHsDevice_DisplayedStatus) now contains the appropriate data. Because a device status is based on the features, these properties will be blank for devices. Read the features to get the different statuses.
